snippets/ibu-ApplicationClusterScopedBackupRestore.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  annotations:
+    lca.openshift.io/apply-label: "apiextensions.k8s.io/v1/customresourcedefinitions/test.example.com,security.openshift.io/v1/securitycontextconstraints/test,rbac.authorization.k8s.io/v1/clusterroles/test-role,rbac.authorization.k8s.io/v1/clusterrolebindings/system:openshift:scc:test"
+  name: backup-app-cluster-resources
+  labels:
+    velero.io/storage-location: default
+  namespace: openshift-adp
+spec:
+  includedClusterScopedResources:
+  - customresourcedefinitions
+  - securitycontextconstraints
+  - clusterrolebindings
+  - clusterroles
+  excludedClusterScopedResources:
+  - Namespace
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: test-app-cluster-resources
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3"
```

```suggestion
    lca.openshift.io/apply-wave: "3" <2>
```
<2> The wave number should be higher than that specified in the platform Restore CRs and lower than the one in the application namespace-scoped Restore CR.